src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,32 @@
+package calculator;
+
+import org.apache.logging.log4j.util.Strings;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class StringCalculator {
+    private final int DEFAULT_INT_VALUE = 0;
```

Comment:
상수는 `static final` 필드입니다. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private static final int DEFAULT_INT_VALUE = 0;
```